requirements.txt

```
@@ -2,8 +2,9 @@ arrow
 ipywidgets
 jupyter
 langchain
+langchain-community
```

the deprecation warning is coming from them moving huggingface into its own module.

```suggestion
langchain-huggingface
```

---

I'm okay if you fix this with your change as we'll need to change the notebook imports

---

It would appear that `langchain-community` is still required for some other things around opensearch.

---

I'll make it require both - I hope order doesn't matter :)